src/Contexts/ThemeContext.tsx

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext, useState, ReactNode  } from "react"
+
```

Comment:
Hey Devin, the circleCI build was failing at the declaration of ThemeContextProps.  I added a comment for eslint to ignore that code block for the sake of being able to merge the code.  We might want to go back and use this interface statement in the future. Everything else looks great!